pkg/cedar/intake/schemas/easi_system_intake.json

```diff
@@ -1,8 +1,38 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
   "$ref": "#/definitions/EASIIntake",
-  "title": "EASIIntakeV04",
+  "title": "EASIIntakeV03",
```

Comment:
Run `scripts/gen_intake_schema` to regenerate the schema with the latest version and all changes made to the model.

Reply:
I'm going to need someone else to generate the intake schema and push it for me - I've got a weird issue that seems to only be affecting me that is preventing me from being able to generate the schema (CEDAR rejects my cert, but only for this specific task). @ClayBenson94 and I were were on a call for a good 45-60 min this morning trying to figure it out with no luck. Will keep noodling on it, but in the meantime it would be helpful to unblock Ashley.

Reply:
I generated the schema and pushed it to this branch, it's now updated.